static/js/stream_edit.js

```diff
@@ -591,6 +594,13 @@ exports.initialize = function () {
     $("#subscriptions_table").on("submit", ".subscriber_list_add form",
                                  submit_subscriber_form);
 
+    $("#subscriptions_table").on("keydown", ".subscriber_list_add", function (e) {
+        if (e.keyCode === 13) {
+            submit_subscriber_form(e);
+        }
+        return;
```

Comment:
I don't think we need a `return` here.

Reply:
Not actually. I will remove it..thank you for your feedback.